MRV Overhaul

MRVDOG opened the pull request.

Commit:
MRV Overhaul

Commit:
CSS tweak (background inherits)

Commit:
FIX: global `font-family`

Commit:
Changed date format

ProbablePrime commented on assets/index.js:
```diff
@@ -0,0 +1,84 @@
+  }
+
+  setTimes() {
+    this.now = moment.tz("America/Los_Angeles");
```
https://github.com/you-dont-need/You-Dont-Need-Momentjs

`toLocaleTimeString()` already did what we needed here.

ProbablePrime reviewed (changes requested):
Strip out moment and we're good :)

Commit:
Removed Momentjs - now 100% native date

I really aren't a big fan of native dates, I usually use Luxon but that wasn't on CF's CDN, which is why I went with moment, didn't realize how large it was :P it was a pain to get working, but I managed :)

Commit:
Fix: remaining time ended in comma if mins were removed (at 0)

Commit:
FIX: lower cased am/pm

Commit:
Fix: removed delay for info to load in
